markdown templates/virtual-machines-ps-template-compare-sm-arm-task.md

```diff
@@ -27,33 +27,33 @@ If you haven't done so already, install at least the [version number] version of
 ## Command Comparison
 This [table | section] shows the command syntax.
 
-<!--[optional image - to use an image in this article, add a folder with the same name as the article file name without extension, inside the Media folder of the repo. Use only this folder to store the images. Don't attempt to use a common folder to share images you want to use in more than 1 file.]
+<!--[optional image - to use an image in this article, add a folder with the same name as the article file name without extension inside the Media folder of the repo. Use only this folder to store the images. Don't attempt to use a common folder to share images you want to use in more than 1 file.]
 Then, use the following syntax to add a reference to the image in your article:
 ![](./media/name-of-file-without-extension/image-name-no-spaces.png)
 -->
 
-<!--if a command string uses variables, define the variables first, using the  following construction. In some cases the variable is straightforward and doesn't need much explanation. But parameters such as location and size can benefit from brief explanation or listing all accepted values:--> 
+<!--if a command string uses variables, define the variables first using the following construction. Sometimes, the variable is straightforward and doesn't need much explanation. But parameters such as location and size can benefit from brief description or listing of all accepted values:--> 
 
 These command examples use the following variables:
 
 $FriendlyName"\<Describe value>"
 
-<!-- if it makes more sense to present this in a table, use this. Otherwise, delete. The table won't render until it's in GitHub or published to Sandbox.-->
+<!-- if it makes more sense to present this in a table, use this. Otherwise, delete. The table will render once it's in GitHub or published to Sandbox.-->
```

Comment:
```suggestion
<!-- if it makes more sense to present this in a table, use this. Otherwise, delete. The table will render once it's in GitHub or published to Sandbox.-->
```
```suggestion
<!-- if it makes more sense to present this in a table, use this. Otherwise, delete. The table will render after it's in GitHub or published to Sandbox.-->
```